backend/census_historical_migration/sac_general_lib/general_information.py

```diff
@@ -0,0 +1,147 @@
+import audit.validators
+from datetime import date, timedelta
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+)
+from census_historical_migration.sac_general_lib.utils import (
+    _census_date_to_datetime,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+    _boolean_field,
+)
+
+mappings = [
+    FormFieldMap(
+        "auditee_fiscal_period_start", "fyenddate", "fy_start_date", None, str

```

Comment:
It is possible (perhaps not in this commit, but soon) that we want an xform that replaces all foundational Python types.

```
def string_to_string(...):
    # error checking, normalization
    # throw an exception if some condition happens
    return str(input)
```

or similar.

This way, we have a centralized xform for all strings, all bools, etc.

I'm particularly thinking about the email question, below. It might want its own special transform, to handle the missing case. (Or, perhaps the missing case/exception/error is part of a standard string-to-string transform).

Reply:
I created a [ticket](https://github.com/GSA-TTS/FAC/issues/2833) for this to limit the size of the PR

Reply:
Actually, I saw a need for it and implemented this method and few others as part of the current PR. The ticket mentioned above will be use to improve that `string_to_string` method and add others (if necessary)